src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure implements Drawable, AreaCalculable {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(double firstLeg, double secondLeg, String color) {
+        super.setColor(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = firstLeg;
```

Comment:
It seems like there's a typo here. You're assigning 'firstLeg' to 'secondLeg'. This should be 'this.secondLeg = secondLeg;' to correctly assign the passed 'secondLeg' parameter to the 'secondLeg' field.